src/openforms/submissions/tests/test_tasks_pdf.py

```diff
@@ -136,7 +136,7 @@ def test_date_object_is_converted_to_str_when_it_comes_from_logic_rule(self):
                     "variable": "updatedDate",
                     "action": {
                         "type": "variable",
-                        "value": {"+": [{"var": "date1"}, {"duration": "P1M"}]},
+                        "value": {"+": [{"var": "date1"}, {"duration": "P15D"}]},
```

Comment:
I'm not sure how this test didn't fail on master, adding `P1M` (31 days) to `2025-01-01` should result in `02-01-2025`, which it did on my branch (https://github.com/open-formulieren/open-forms/actions/runs/13111391312/job/36575912944?pr=5036#step:7:128)